provision-contest/disable-turboboost_ht

```diff
@@ -22,7 +22,7 @@ for cpu in $(ls -1d /sys/devices/system/cpu/cpu* | sort --version-sort) ; do
 	fi
 
 	# Disable all but one thread on each core.
-	core_id=$(cat $cpu/topology/core_id)
+	core_id=$(cat $cpu/topology/core_id $cpu/topology/physical_package_id)
```

Comment:
I think this should be separated by an underscore or so. Now there's a (small) risk of collisions: core 11 on physical_package 1 and core 1 on phyiscal package 11 would be considered identical.

Reply:
Good catch!

Reply:
Good point, which I should've added as a comment. `cat` already adds a newline after each file so this cannot happen. Will add the comment and/or add `| tr '\n' '_'` or similar to make it even more explicit.